source/upgrade.txt

```diff
@@ -76,6 +76,16 @@ Version 6.x Breaking Changes
 - Automatic Encryption methods are now in the {+driver-short+}. You must
   import these methods from the driver instead of from
   ``mongodb-client-encryption``.
+- Removed the ``ObjectId`` constructor that accepted a 12-character string.
+- The ``abortTransaction()`` and ``commitTransaction()`` methods return
```

Comment:
Suggestion:

I think it could be helpful to match the format of the other list items by starting it with a verb.

```suggestion
   - Modified ``abortTransaction()`` and ``commitTransaction()`` methods to return
```